src/main/java/org/broadinstitute/hellbender/engine/ProgressMeter.java

```diff
@@ -181,11 +187,25 @@ public void setRecordLabel( final String label ) {
         this.recordLabel = label;
     }
 
+    /**
+     * Set whether this ProgressMeter should be disabled. When disabled, future calls to start(), stop(),
+     * and update() have no effect.
+     *
+     * @param disabled if true, disable this ProgressMeter
+     */
+    public void setDisabled(final boolean disabled) {
```

Comment:
The fact that this is mutable lets you put it into weird states that don't really make sense.  Like started but disabled.    
I think this should be moved into a constructor argument at a minimum.  If it is a constructor argument though it's still kind of bizarre to have an if statement before every function that skips them.  I think ideally we'd have a NoOp progress meter as a separate class instead instead of making this one more complicated.  We could pull out a progress meter interface and have a no-op implementation instead.